### PR TITLE
Add Firefox compatibility

### LINKED ISSUE #90: Plugin errors after update to Chrome 60
I started receiving the following error in the dev tools after  chrome ver 60 update.

image

chrome version

parity ext issues

### PR DESCRIPTION
**Pull request by tomusdrw**
Related #87 
Closes #90

### CODE REVIEW

File: web3/inpage.js [hunk 1]
```diff
@@ -25,11 +26,9 @@ import Web3FrameProvider from './provider';
 import './embed.html';
 
 // Indicate that the extension is installed.
-window[Symbol.for('parity.extension')] = {
-  version: require('../package.json').version
-};
+setInstalled();
 
-if (!window.chrome || !window.chrome.extension) {
+if (!window.browse || !window.browse.extension) {
```
Comment (ngotchac) [inline web3/inpage.js]: `browser` ?

**Review by ngotchac — COMMENTED** [review]
That's great ! A tiny question though

File: web3/inpage.js (hunk 1 above)
Comment (tomusdrw) [reply web3/inpage.js]: Great catch!

Comment (ngotchac) [reply web3/inpage.js]: (plus I guess check for `chrome` Object)

Comment (tomusdrw) [reply web3/inpage.js]: Fixed.

**Review by ngotchac — APPROVED** [review]

**Pull request merged**